src/components/RegisterCard/RegisterCard.js

```diff
@@ -14,6 +13,21 @@ import addPhoto from "../../addPhoto";
 import MESSAGES from "../../constants/messages";
 import METHODS from "../../constants/methods";
 import NAMES from "../../constants/names";
+import VARIANTS from "../../constants/variants";
+
+const {
+  FAILED_UPLOAD_IMAGE,
+  OK
```

Comment:
Trailing commas 누락되어 있습니다!

```suggestion
  OK,
```